<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateSelectDialog</class>
 <widget class="QDialog" name="TemplateSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择模板</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string>font-weight: bold; font-size: 14px;</string>
     </property>
     <property name="text">
      <string>请选择模板或输入模板号:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>
       border: 1px solid #ccc;
       border-radius: 5px;
       padding: 4px;
       font-size: 13px;
      </string>
     </property>
     <property name="placeholderText">
      <string>输入模板号或扫码...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="tableWidgetTop5">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>
       QTableWidget {
           background-color: #f9f9f9;
           border: 1px solid #ccc;
           font-size: 13px;
       }
       QTableWidget::item:selected {
           background-color: #87cefa;
           color: black;
           font-weight: bold;
       }
      </string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>
       QDialogButtonBox QPushButton {
           min-width: 80px;
           padding: 5px;
           border-radius: 5px;
           background-color: #5cb85c;
           color: white;
           font-weight: bold;
       }
       QDialogButtonBox QPushButton:disabled {
           background-color: #aaa;
       }
      </string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
